packages/comet-uswds/src/components/form/form.tsx

```diff
@@ -25,7 +25,6 @@ export const Form = ({
     <form
       id={id}
       className={classNames('usa-form', { 'usa-form--large': isLarge }, className)}
-      style={{ maxWidth: 'unset' }}
```

Comment:
This was something I added at some point. It really doesn't adhere to USWDS settings, so removing it